packages/framework/src/servers/nest/nest.interface.ts

```diff
@@ -2,4 +2,5 @@ import type { ServeHandlerOptions } from '../../handler';
 
 export type NovuModuleOptions = ServeHandlerOptions & {
   apiPath: string;
+  controllerDecorators?: ClassDecorator[];
```

Comment:
This allows customers to add custom decorators to the controller and fixed the swagger conflict.

Reply:
Awesome!